client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -430,27 +430,35 @@ class PluginMeta extends Component {
 	}
 
 	render() {
+		const {
+			selectedSite,
+			plugin: pluginProp,
+			isPlaceholder,
+			sites,
+			atEnabled,
+			slug,
+			translate,
+			isMock,
+		} = this.props;
 		const cardClasses = classNames( 'plugin-meta__information', {
 			'has-button': this.hasOrgInstallButton(),
-			'has-site': !! this.props.selectedSite,
-			'is-placeholder': !! this.props.isPlaceholder
+			'has-site': !! selectedSite,
+			'is-placeholder': !! isPlaceholder
 		} );
-
-		const plugin = this.props.selectedSite && this.props.sites[ 0 ] && this.props.sites[ 0 ].plugin
-			? this.props.sites[ 0 ].plugin
-			: this.props.plugin;
-		const path = ( ! this.props.selectedSite || plugin.active ) && this.getExtensionSettingsPath( plugin );
+		const selectedSitePlugin = get( sites, '[0].plugin', pluginProp );
+		const plugin = selectedSite ? selectedSitePlugin : pluginProp;
```

Comment:
This says, "if `selectedSite` is a truthy value, then use `selectedSitePlugin` for the value of `plugin`". `selectedSitePlugin` could be false (if we don't have a truthy value for `this.props.sites[ 0 ].plugin`), which puts us in the same possible error states we try to avoid in the previous PR. Instead, we could do:

```js
const selectedSitePlugin = get( sites, '[0].plugin', orgPlugin );
const plugin = selectedSite && selectedSitePlugin ? selectedSitePlugin : orgPlugin;
```